src/legacy/core_plugins/data/public/search/search_bar/components/search_bar.tsx

```diff
@@ -28,7 +28,7 @@ import { FilterBar } from 'ui/filter_bar';
 import { IndexPattern } from 'ui/index_patterns';
 import { Storage } from 'ui/storage';
 
-import { QueryBar } from '../../query_bar';
+import { QueryBar } from '../../../query/query_bar';
```

Comment:
nit: I still think we should eventually consider passing this from `Query` to `Search` via the `setup` method, but that discussion is outside of the scope of this PR so let's sort it out later.

Reply:
Agreed